junit-runtime/src/main/scala/com/novocode/junit/OutputCapture.scala

```diff
@@ -0,0 +1,47 @@
+package com.novocode.junit
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+object OutputCapture {
+  def start(): OutputCapture = new OutputCapture()
+
+  private def getScalaOut(): PrintStream = {
+    try {
+      scala.Console.out
+    } catch {
+      case t: Throwable => null
```

Comment:
I have no idea. There I'm just mimicking the behaviour of the original jUnit interface ([original code](https://github.com/sbt/junit-interface/blob/master/src/main/java/com/novocode/junit/OutputCapture.java#L41-L55)).